src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl

```diff
@@ -111,6 +111,34 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template match="note">
+    <xsl:variable name="type">
+      <xsl:value-of select="normalize-space(@type)"/>
+    </xsl:variable>
+
+    <div>
+      <xsl:attribute name="class">
+        <xsl:choose>
+          <xsl:when test="$type = 'warning'">WARNING alert alert-warning</xsl:when>
+          <xsl:when test="$type = 'caution'">CAUTION alert alert-danger</xsl:when>
+          <xsl:when test="$type = 'important'">IMPORTANT alert alert-danger</xsl:when>
+          <xsl:otherwise>NOTE alert alert-info</xsl:otherwise>
+        </xsl:choose>    
+      </xsl:attribute>
+      <h5>
```

Comment:
this is the default behavior of DFM, but user can change it by [token.json](https://github.com/dotnet/docfx/blob/dev/src/docfx.website.themes/default/token.json#L39)